test.js

```diff
@@ -1,6 +1,8 @@
 const { describe, beforeEach, it } = require('mocha')
-const { expect } = require('chai')
+const { expect, assert } = require('chai')
```

Comment:
Couldn't get `expect().to.be` working, kept failing `.to` or `.be` is not a function.

Reply:
That's odd, but I tend to prefer `assert` any ways, mainly pulled in `expect` for its `throw` syntax ...

Perhaps we could switch to [Node's built in throw helper](https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message) and drop our use of `expect` entirely, using `chai` only for the snapshots?

Reply:
TIL node has its own assert 😄